### PR TITLE
upgrade to ember cli 2.5.0

### PR DESCRIPTION
**Pull request by toddjordan**

### CODE REVIEW

**Comment by toddjordan** [comment]
cc @locks @acorncom

File: bower.json [hunk 1]
```diff
@@ -1,16 +1,9 @@
 {
   "name": "super-rentals",
   "dependencies": {
-    "ember": "2.3.0",
-    "ember-cli-shims": "0.1.0",
+    "ember": "~2.5.0",
+    "ember-cli-shims": "0.1.1",
     "ember-cli-test-loader": "0.2.2",
-    "ember-qunit-notifications": "0.1.0",
-    "jquery": "1.11.3",
```
Comment (locks) [inline bower.json:8]: shouldn't `jquery` stay?

Comment (toddjordan) [inline bower.json:8]: Interestingly, its no longer there by default.  https://github.com/ember-cli/ember-new-output/compare/v2.4.3...v2.5.0

**Comment by toddjordan** [comment]
currently looking at the build failure: 
image

**Comment by toddjordan** [comment]
changes compile and run

**Pull request merged**